gulpfile.js

```diff
@@ -0,0 +1,68 @@
+var fs = require('fs');
+var pkg = JSON.parse(fs.readFileSync('./package.json', 'utf8'));
+var gulp = require('gulp');
+var sass = require('gulp-ruby-sass');
+var strip = require('gulp-remove-code');
+var header = require('gulp-header');
+var uglify = require('gulp-uglify');
+var rename = require('gulp-rename');
+var browserSync = require('browser-sync');
+var reload = browserSync.reload;
+var del = require('del');
+
+var buildDir = 'dist';
+var banner = '/*!\n' +
+        ' * Validetta (<%= pkg.homepage %>)\n' +
+        ' * Version <%= pkg.version %>\n'+
+        ' * Licensed under <%= pkg.license.type %> (<%= pkg.license.url %>)\n' +
+        ' * Copyright 2013-<%= new Date().getFullYear() %> <%= pkg.author.name %> - <%= pkg.author.url %> \n' +
+        ' */\n';
+
+
+/* development tasks */
```

Comment:
I notice you like the lower case comments here. As a heading though, I feel like we should capitalize?